ftest/src/test/java/org/jboss/arquillian/graphene/ftest/utils/GrapheneSelect.java

```diff
@@ -0,0 +1,340 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2013, Red Hat, Inc. and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.arquillian.graphene.ftest.utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.WrapsElement;
+import org.openqa.selenium.support.ui.ISelect;
+import org.openqa.selenium.support.ui.Quotes;
+import org.openqa.selenium.support.ui.UnexpectedTagNameException;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.StringTokenizer;
+import java.util.stream.Collectors;
+
+/**
+ * Temporary class for GrapheneSelect elements
+ * Replace with <code>{@link org.openqa.selenium.support.ui.Select}</code> after resolving issues with getDomProperty() and getDomAttribute() methods
+ * <p>
+ * Models a SELECT tag, providing helper methods to select and deselect options.
+ */
+public class GrapheneSelect implements ISelect, WrapsElement {
```

Comment:
This class was added to the codebase due to several web drivers don't implement `getDomAttribute()` for `WebElement` class and the `UnsupportedOperationException` is thrown then. I also tried to update WebDrivers versions, but unsuccessfully. I've noticed the issue is quite well-known in Selenium community. See the Selenium issue [#8995](https://github.com/SeleniumHQ/selenium/issues/9895).

ChromeDriver version 96.0.4664.45
Gecko driver: 0.30.0